docs/howto/pipeline_testing.md

```diff
@@ -81,30 +102,12 @@ The pipeline test runner transforms log lines into file input events and sends t
 }
 ```
 
-The `multiline` section configures the log file reader to correctly detect multiline log entries using the `first_line_pattern`. Use this property if your logs may be split into multiple lines, e.g. Java stack traces.
+The `multiline` section (raw files only) configures the log file reader to correctly detect multiline log entries using the `first_line_pattern`. Use this property if your logs may be split into multiple lines, e.g. Java stack traces.
```

Comment:
Maybe link `raw files` to the "Raw files" section above?

Reply:
Raw files linked.